### PR TITLE
bridge_with_irc: Make bridge more user-friendly.

### PR DESCRIPTION
**Pull request by orientor**
Edit error and success messages of zulip-irc bridge to be more
user friendly. Also check whether zulip bot is subscribed before
starting irc bot.

### CODE REVIEW

**Comment by orientor** [comment]
@rht @showell changed the messages displayed on running irc-bot. Kindly review.

File: zulip/integrations/bridge_with_irc/irc_mirror_backend.py [hunk 1]
```diff
@@ -21,6 +20,8 @@ def __init__(self, zulip_client, stream, topic, channel,
         self.nickserv_password = nickserv_password
         # Make sure the bot is subscribed to the stream
         self.check_subscription_or_die()
+        # Connect to bot after proper connection to Zulip server has been confirmed.
+        irc.bot.SingleServerIRCBot.__init__(self, [(server, port)], nickname, nickname)
```
Comment (rht) [inline zulip/integrations/bridge_with_irc/irc_mirror_backend.py]: This change should probably in its own commit.

File: zulip/integrations/bridge_with_irc/irc_mirror_backend.py [hunk 2]
```diff
@@ -35,15 +36,18 @@ def connect(self, *args, **kwargs):
         self.reactor.loop.run_until_complete(
             self.connection.connect(*args, **kwargs)
         )
-        print("Connected to IRC server.")
+        print("Connected to IRC server. The bot has started.")
```
Comment (rht) [inline zulip/integrations/bridge_with_irc/irc_mirror_backend.py]: The message that @showell suggested in czo was different, wasn't it. Though I am fine with this one as well.

**Comment by rht** [comment]
The commit message has redundant description of "bridge_with_irc" and "zulip-irc bridge".

File: zulip/integrations/bridge_with_irc/irc_mirror_backend.py (hunk 1 above)
Comment (orientor) [reply zulip/integrations/bridge_with_irc/irc_mirror_backend.py]: I have split the commit in two.

File: zulip/integrations/bridge_with_irc/irc_mirror_backend.py (hunk 2 above)
Comment (orientor) [reply zulip/integrations/bridge_with_irc/irc_mirror_backend.py]: Changed the message.

**Comment by showell** [comment]
@rht Once you're happy with this, I'll merge it.  Everything here seems pretty reasonable to me.

@orientor Thanks for the quick turnaround here!

**Comment by showell** [comment]
@rht For the redundancy bit, what's your suggested edit to the commit message?

**Comment by rht** [comment]
For the commit message, I was thinking of along the line of "bridge_with_irc: Improve user-friendliness"; everything else LGTM.

**Comment by orientor** [comment]
Changed the commit message.

**Pull request merged**